IdokladSdk.IntegrationTests/Tests/Clients/DocumentPayment/SalesDocumentPaymentTests.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using IdokladSdk.Clients;
+using IdokladSdk.Enums;
+using IdokladSdk.IntegrationTests.Core;
+using IdokladSdk.IntegrationTests.Core.Extensions;
+using IdokladSdk.Requests.Core.Modifiers.Filters.Common;
+using NUnit.Framework;
+
+namespace IdokladSdk.IntegrationTests.Tests.Clients.DocumentPayment
+{
+    /// <summary>
+    /// SalesDocumentPaymentTests.
+    /// </summary>
+    [TestFixture]
+    public class SalesDocumentPaymentTests : TestBase
+    {
+        public DocumentPaymentClient DocumentPaymentClient { get; set; }
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            InitDokladApi();
+            DocumentPaymentClient = DokladApi.DocumentPaymentClient;
+        }
+
+        [Test]
+        public async Task GetGetCreditNotePayment_ReturnsPayment()
+        {
+            // Arrange
+            var page = await DocumentPaymentClient.Sales.List().PageSize(1)
+                .GetAsync(c => new { c.Id }, DocumentPaymentSalesType.CreditNote).AssertResult();
+            var id = page.Items.First().Id;
```

Comment:
upravit testy na konkrétní ids a rozšířit asserty

Reply:
rozšířeno